features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineView.kt

```diff
@@ -232,6 +256,36 @@ fun TimelineItemEventRow(
     }
 }
 
+@Composable
+private fun TimestampView(
+    sentTime: String,
+    isEdited: Boolean,
+    sendingFailed: Boolean,
+    onResendClicked: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val tint = if (sendingFailed) ElementTheme.colors.textActionCritical else null
+    Row(modifier = modifier.clickable(enabled = sendingFailed, onClick = onResendClicked)){
+        if (isEdited) {
+            Text(
+                stringResource(StringR.string.common_edited_suffix),
+                style = ElementTextStyles.Regular.caption2,
+                color = tint ?: MaterialTheme.colorScheme.secondary,
+            )
```

Comment:
(Quick note: this will be done in the future, but clicking on this text should open the edition history.)

Reply:
There might be a corner case where we have an edited message that couldn't be send, maybe? I've asked what we should do in this case too.